### PR TITLE
Update notification box styling, content

### LINKED ISSUE #13992: strange styling in "Notifications" popup
## Description

When clicking the "Notifications" bell on the bottom right, the resultant popup lacks a border and shadow to distinguish it from the background. This hinders usability of the popup as it is not immediately obvious where the popup ends.

Screen Shot 2023-02-13 at 2 41 45 PM

## Reproduce

1. Open JupyterLab and click on the bottom right notifications icon.

## Expected behavior

The notifications popup should have a border and shadow to distinguish it from the background.

## Context

- Operating System and version: OS X Monterey
- Browser and version: Chrome 109.0.5414.119 (Official Build) (x86_64)
- JupyterLab version: [4.0.0a33 binder](https://gist.github.com/jtpio/4e9a3d435d7c3950dc4150c10abbb1d7)

### PR DESCRIPTION
**Pull request by JasonWeill**
## References

Fixes #13992 and #13993.

## Code changes

Updates the content of the "attempting to save read-only file" notification to match Jupyter styling with copy edits. Updates the CSS to put a small border around a notification display and to modify the padding for the header.

Adds another newline to the Jupyter news notification that appears on first startup, providing additional separation between the question and a clarifying request.

## User-facing changes

Before the change:

Screenshot 2024-04-24 at 3 41 17 PM

Screenshot 2024-04-24 at 3 58 11 PM

After the change:

Screenshot 2024-04-25 at 10 05 45 AM

Screenshot 2024-04-25 at 10 05 32 AM

## Backwards-incompatible changes

None.

### CODE REVIEW

File: packages/docregistry/src/components.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export function createReadonlyLabel(
       <span
         className="jp-ToolbarLabelComponent"
         title={trans.__(
-          `Document is permissioned read-only; "save" is disabled, use "save as..." instead`
+          `Document is read-only. "Save" is disabled; use "Save as…" instead`
```
Comment (chanhee-park) [inline packages/docregistry/src/components.tsx:26]: Why use semicolons? Wouldn’t the punctuation be neater? 
```
`Document is read-only. "Save" is disabled. Use "Save as…" instead`
```

File: packages/apputils-extension/src/announcements.ts [hunk 2]
```diff
@@ -100,7 +100,7 @@ export const announcements: JupyterFrontEndPlugin<void> = {
       if (mustFetchNews === 'none') {
         const notificationId = Notification.emit(
           trans.__(
-            'Would you like to receive official Jupyter news?\nPlease read the privacy policy.'
+            'Would you like to receive official Jupyter news?\n\nPlease read the privacy policy.'
```
Comment (krassowski) [inline packages/apputils-extension/src/announcements.ts:103]: Some thoughts:
- there are numerous complaints about this popup. This change will make it even larger.
- do we need the "Please read the privacy policy" line at all? There is a link to privacy policy below anyways.
- "receive" is correct but for me it has a connotation of something more heavy weight (e.g. receive (e)mail). I feel like "get notified of" might fit here better?

```suggestion
            'Would you like to get notified official Jupyter news?'
```

**Comment by krassowski** [comment]
The added border looks great!

File: packages/docregistry/src/components.tsx (hunk 1 above)
Comment (JasonWeill) [reply packages/docregistry/src/components.tsx:26]: I'm not sold that this is necessarily neater. The semicolon links two closely related statements, so I think it is still appropriate to keep it in this notification.

**Review by krassowski — DISMISSED** [review]
Thank you @JasonWeill! Good to merge once green.

**Review by krassowski — APPROVED** [review]
Should be good now :)

**Pull request merged**